### ISSUE TITLE
Run Test --- Problem

### ISSUE THREAD

**Issue by christianbender** [abranhe/python-lib#21]
@abranhe How do you run the tests?I tried the following command
```$
python -m unittest discover tests
```
But I got this error message:  

![screenshot](https://user-images.githubusercontent.com/23243382/53287603-bc88ce00-377e-11e9-92bd-f745e74b7efc.png)

I changed the name of the imported module, but got the error message again.

**Comment by christianbender** [comment]
@abranhe The follwing command give me the same error message:  
```
python3 -m unittest tests.test_sorting
```
![screenshot](https://user-images.githubusercontent.com/23243382/53287715-5f8e1780-3780-11e9-97dd-7f35b166f144.png)

**Comment by abranhe** [comment]
Closed on #22
